frontend/src/pages/Help/PDFViewer/PDFViewerContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
```

Comment:
Maybe this logic will be common for all the PDF inside our application? There why it will be better (maybe?) to move all the logic to the PDFViewer container inside common components

Reply:
Exactly